### PR TITLE
added the ability to log body

### PR DESCRIPTION
**Pull request by allandegnan**
Defaults to not logging it.

### CODE REVIEW

File: go.sh [hunk 1]
```diff
@@ -103,6 +103,10 @@ case "${LOG_FORMAT_NAME}" in
             sed -i -e 's/\$request_uri/\$uri/g' ${NGIX_CONF_DIR}/logging.conf
         fi
 
+        if [ "${NO_LOGGING_BODY" ]; then
```
Comment (gambol99) [inline go.sh]: bracket

**Comment by gambol99** [comment]
@lewismarshall @vaijab

File: README.md [hunk 2]
```diff
@@ -67,6 +67,7 @@ managed by Docker or Kubernetes.
 This is useful when testing or for development instances or when a load-balancer mandates a non-standard port.
 * `LOG_FORMAT_NAME` - Can be set to `text` or `json` (default).
 * `NO_LOGGING_URL_PARAMS` - Can be set to `TRUE` if you don't want to log url params. Default is empty which means URL params are logged
+* `NO_LOGGING_BODY` - Defaults to true `TRUE`.  Set otherwise and nginx should log the request_body.
```
Comment (vaijab) [inline README.md]: Would be great to change it to `LOG_BODY=FALSE`, but I see that `NO_LOGGING_URL_PARAMS` is there already. This is similar to double negatives in English, which is very confusing :-) /cc @timgent

**Comment by vaijab** [comment]
LGTM otherwise.

File: README.md (hunk 2 above)
Comment (allandegnan) [inline README.md]: I agree with this but I'm going to fix both of them together in a future PR.

**Pull request merged**